pkg/bridge/connectors/modulr/errors.go

```diff
@@ -0,0 +1,14 @@
+package modulr
+
+import "github.com/pkg/errors"
+
+var (
+	// ErrMissingTask is returned when the task is missing.
+	ErrMissingTask = errors.New("task is not implemented")
```

Comment:
I think this can be moved to the framework part.

Reply:
Will be fixed in next PR